City-of-Helsinki/drupal-infofinland
# IFU-866 - removed fields, data and anything related to PTV

# [IFU-866](https://helsinkisolutionoffice.atlassian.net/browse/IFU-866)
The PTV module was already removed, but this PR should also remove the fields and associated data. It is a rather heavy PR.

## What was done


## How to install

* Make sure your instance is up and running on correct branch.
  * `git checkout feature/IFU-866-remove-ptv`
  * `make fresh`
* Run `make drush-cr`

## How to test
Not sure how to test this, except to verify there is no evidence of PTV data on content or related fields.



[IFU-866]: https://helsinkisolutionoffice.atlassian.net/browse/IFU-866?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
